docs/empty-package-output.md

```diff
@@ -0,0 +1,16 @@
+# Empty package in Embroider output
+
+As part of Embroider build process traditional (v1) addons are rewritten to the new v2 format before building your application. Embroider can only tell what contents those packages should have by letting the traditional broccoli-based build pipelines run and observing the files that come out but this can be a bit tricky because there are some cases where broccoli won't run for a particular addon. 
+
+There are several reasons an addon won't ever get consumed by broccoli, causing embroider to emit an empty package. Some of those reasons include:
+
+- if an app uses `addons.blacklist`, the excluded addon can end up empty since nobody consumed it
```

Comment:
```suggestion
- if an app uses `addons.blacklist` or `addons.exclude`, the excluded addon can end up empty since nobody consumed it
```

`addons.blacklist` will be removed in `ember-cli` v5.

Reply:
I've included this change 👍